src/controllers/trigger/common/trigger.js

```diff
@@ -14,6 +14,36 @@ import { internalServerError, notFound, ok } from '@adobe/spacecat-shared-http-u
 
 import { isAuditForAllUrls, isAuditForAllDeliveryTypes, sendAuditMessages } from '../../../support/utils.js';
 
+function isAllAuditsDisabled(site, organization) {
+  const orgAuditConfig = organization.getAuditConfig();
+  const siteAuditConfig = site.getAuditConfig();
+
+  return orgAuditConfig.auditsDisabled() || siteAuditConfig.auditsDisabled();
+}
+
+function isAuditTypeDisabled(auditType, site, organization) {
+  const orgAuditDisabled = organization.getAuditConfig().getAuditTypeConfig(auditType)?.disabled();
+  const siteAuditDisabled = site.getAuditConfig().getAuditTypeConfig(auditType)?.disabled();
+
+  return orgAuditDisabled || siteAuditDisabled;
+}
```

Comment:
This logic will probably be heavily re-used in audit-worker and others, should we centralize?

Reply:
please see: https://github.com/adobe/spacecat-shared/pull/173